.github/workflows/ci_pipeline.yml

```diff
@@ -77,6 +97,13 @@ jobs:
     - name: Build Executable
       run: |
         poetry run pyinstaller --add-data "cover_agent/prompt_template.md:." --add-data "cover_agent/version.txt:." --hidden-import=tiktoken_ext.openai_public --hidden-import=tiktoken_ext --onefile --name cover-agent-${{ matrix.os }} cover_agent/main.py
```

Comment:
```suggestion
      run: |
        poetry run pyinstaller --add-data "cover_agent/prompt_template.md:." --add-data "cover_agent/version.txt:." --add-data "$(python -c 'import litellm, os; print(os.path.join(os.path.dirname(litellm.__file__), "llms", "tokenizers"))'):litellm/llms/tokenizers" --hidden-import=tiktoken_ext.openai_public --hidden-import=tiktoken_ext --onefile --name cover-agent-${{ matrix.os }} cover_agent/main.py
```
@EmbeddedDevops1  disclaimer: I'm not a python expert - we can all the tokenizers from site-packages. 

```
"$(python -c 'import litellm, os; print(os.path.join(os.path.dirname(litellm.__file__), "llms", "tokenizers"))'):litellm/llms/tokenizers"
```